types/enzyme/tsconfig.json

```diff
@@ -5,6 +5,14 @@
             "es6",
             "dom"
         ],
+        "paths": {
+            "react": [
+                "react/v16"
```

Comment:
It should test against every version of react prior to 17, not just 16

Reply:
DT has no infra for that I believe. This is mostly for the publisher to ensure it doesn't pull in v17 or later types.